### ISSUE TITLE
QR code for Maiar connection is shown just the first time

### ISSUE THREAD

**Issue by joaquin-alfaro** [ElrondNetwork/dapp-core#89]
When using the components _UnlockPage_, _WalletConnectLoginButton_ or _WalletConnectLoginContainer_ the QR code for Maiar connection is just shown the first time it is asked, next times it is shown empty.

### Expected behaviour
When using the components _UnlockPage_, _WalletConnectLoginButton_ or _WalletConnectLoginContainer_ the QR code for Maiar connection to be shown even if it has been generated once.

### Actual behaviour
The QR code for Maiar connection is shown just the first time it is asked, next times it is shown empty.

### Steps to reproduce the behavior
1. Add dependency with dapp-core@**0.4.6** to [dapp-template](https://github.com/ElrondNetwork/dapp-template)
```
 "dependencies": {
   "@elrondnetwork/dapp-core": "0.4.6"
}
```
2. Start _dapp-template_, navigate to the page _/unlock_ and click **Maiar** button. It will be shown the QR code to establish connection with Maiar App.
3. Close the modal with the QR code and click **Maiar** button again, the QR codes is not shown.

### Cause of the issue
The release **0.4.6** introduced the following change in [useWalletConnectLogin.tsx](https://raw.githubusercontent.com/ElrondNetwork/dapp-core/master/src/services/login/useWalletConnectLogin.ts) 

```
  if (!walletConnectBridge || providerRef?.current?.isInitialized?.()) {
    return;
  }
```
It seems that the condition **providerRef?.current?.isInitialized?.()** avoids the calling to the function _generateWcUri()_ to generate the _walletConnectUri_ used to generate the QR code.

Is it possible to call **generateWcUri()** even if providerRef is initialized?

**Comment by StanislavSava** [comment]
This is fixed in 1.0.7

**Comment by PaulWCZ** [comment]
It works well !
Thanks
